idx/memory/memory.go

```diff
@@ -16,20 +16,29 @@ import (
 )
 
 var (
+	// metric idx.memory.update.ok is the number of successful updates to the memory idx
+	statUpdateOk = stats.NewCounter32("idx.memory.update.ok")
 	// metric idx.memory.add.ok is the number of successful additions to the memory idx
-	idxOk = stats.NewCounter32("idx.memory.add.ok")
+	statAddOk = stats.NewCounter32("idx.memory.add.ok")
 	// metric idx.memory.add.fail is the number of failed additions to the memory idx
-	idxFail = stats.NewCounter32("idx.memory.add.fail")
-	// metric idx.memory.add is the duration of (successfull) memory idx additions
-	idxAddDuration = stats.NewLatencyHistogram15s32("idx.memory.add")
-	// metric idx.memory.get is the duration of memory idx gets
-	idxGetDuration = stats.NewLatencyHistogram15s32("idx.memory.get")
+	statAddFail = stats.NewCounter32("idx.memory.add.fail")
+	// metric idx.memory.add is the duration of a (successfull) add of a metric to the memory idx
+	statAddDuration = stats.NewLatencyHistogram15s32("idx.memory.add")
+	// metric idx.memory.update is the duration of (successfull) update of a metric to the memory idx
+	statUpdateDuration = stats.NewLatencyHistogram15s32("idx.memory.update")
+	// metric idx.memory.get is the duration of a get of one metric in the memory idx
+	statGetDuration = stats.NewLatencyHistogram15s32("idx.memory.get")
 	// metric idx.memory.list is the duration of memory idx listings
-	idxListDuration = stats.NewLatencyHistogram15s32("idx.memory.list")
+	statListDuration = stats.NewLatencyHistogram15s32("idx.memory.list")
 	// metric idx.memory.find is the duration of memory idx find
-	idxFindDuration = stats.NewLatencyHistogram15s32("idx.memory.find")
-	// metric idx.memory.delete is the duration of memory idx deletes
-	idxDeleteDuration = stats.NewLatencyHistogram15s32("idx.memory.delete")
+	statFindDuration = stats.NewLatencyHistogram15s32("idx.memory.find")
+	// metric idx.memory.delete is the duration of a delete of one or more metrics from the memory idx
+	statDeleteDuration = stats.NewLatencyHistogram15s32("idx.memory.delete")
+	// metric idx.memory.prune is the duration of successful memory idx prunes
+	statPruneDuration = stats.NewLatencyHistogram15s32("idx.memory.prune")
+
+	// metric idx.metrics_active is the amount of currently known metrics in the index
```

Comment:
that might be picky: i think `the number of` is more appropriate than `the amount of` because `metrics` is countable and not a mass of something